go/client/simplefs_windows.go

```diff
@@ -8,89 +8,21 @@ package client
 import (
 	"context"
 	"path/filepath"
-	"strings"
 
 	"github.com/keybase/client/go/libkb"
 	keybase1 "github.com/keybase/client/go/protocol/keybase1"
 )
 
-func doSimpleFSRemoteGlob(g *libkb.GlobalContext, ctx context.Context, cli SimpleFSInterface, path keybase1.Path) ([]keybase1.Path, error) {
-
+func doSimpleFSPlatformGlob(g *libkb.GlobalContext, ctx context.Context, cli SimpleFSInterface, path keybase1.Path) ([]keybase1.Path, error) {
 	var returnPaths []keybase1.Path
-	directory := filepath.ToSlash(filepath.Dir(path.Kbfs()))
-	base := filepath.Base(path.Kbfs())
-	context.TODO()
-
-	// We know the filename has wildcards at this point.
-	// kbfs list only works on directories, so build a glob from a list result.
-
-	g.Log.Debug("doSimpleFSRemoteGlob %s", path.Kbfs())
-
-	opid, err := cli.SimpleFSMakeOpid(ctx)
-	if err != nil {
-		return nil, err
-	}
-	defer cli.SimpleFSClose(ctx, opid)
-
-	err = cli.SimpleFSList(ctx, keybase1.SimpleFSListArg{
-		OpID: opid,
-		Path: keybase1.NewPathWithKbfs(directory),
-	})
-	if err != nil {
-		return nil, err
-	}
 
-	err = cli.SimpleFSWait(ctx, opid)
+	// local glob
+	matches, err := filepath.Glob(path.Local())
 	if err != nil {
 		return nil, err
 	}
-
-	for {
-		listResult, err := cli.SimpleFSReadList(ctx, opid)
-		if err != nil {
-			break
-		}
-		for _, entry := range listResult.Entries {
-			match, err := filepath.Match(base, entry.Name)
-			if err == nil && match == true {
-				returnPaths = append(returnPaths, keybase1.NewPathWithKbfs(filepath.ToSlash(filepath.Join(directory, entry.Name))))
-			}
-		}
-	}
-	return returnPaths, err
-}
-
-func doSimpleFSPlatformGlob(g *libkb.GlobalContext, ctx context.Context, cli SimpleFSInterface, paths []keybase1.Path) ([]keybase1.Path, error) {
-	var returnPaths []keybase1.Path
-	for _, path := range paths {
-		pathType, err := path.PathType()
-		if err != nil {
-			return returnPaths, err
-		}
-
-		pathString := pathToString(path)
-		if strings.ContainsAny(filepath.Base(pathString), "?*") == false {
-			returnPaths = append(returnPaths, path)
-			continue
-		}
-
-		if pathType == keybase1.PathType_KBFS {
-			// remote glob
-			globbed, err := doSimpleFSRemoteGlob(g, ctx, cli, path)
-			if err != nil {
-				return nil, err
-			}
-			returnPaths = append(returnPaths, globbed...)
-		} else {
-			// local glob
-			matches, err := filepath.Glob(pathString)
-			if err != nil {
-				return nil, err
-			}
-			for _, match := range matches {
-				returnPaths = append(returnPaths, keybase1.NewPathWithLocal(match))
-			}
-		}
+	for _, match := range matches {
+		returnPaths = append(returnPaths, keybase1.NewPathWithLocal(match))
```

Comment:
I guess not really, but in practice it will be, right? The shell does the globbing on non-windows. But I guess it can be for all platforms.